src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -55,15 +94,11 @@ internal class MgmtClientOperation : IReadOnlyList<MgmtRestOperation>
         private IReadOnlyList<Parameter>? _methodParameters;
         public IReadOnlyList<Parameter> MethodParameters => _methodParameters ??= EnsureMethodParameters();
 
-        public static MgmtClientOperation FromOperation(MgmtRestOperation operation, Parameter? extensionParameter = null, bool isConvenientOperation = false)
-        {
-            return new MgmtClientOperation(new List<MgmtRestOperation> { operation }, extensionParameter, isConvenientOperation);
-        }
-
         private readonly IReadOnlyList<MgmtRestOperation> _operations;
 
-        private MgmtClientOperation(IReadOnlyList<MgmtRestOperation> operations, Parameter? extensionParameter, bool isConvenientOperation = false)
+        private MgmtClientOperation(MgmtTypeProvider carrier, IReadOnlyList<MgmtRestOperation> operations, Parameter? extensionParameter, bool isConvenientOperation = false)
```

Comment:
What is `carrier`?

Reply:
This `carrier` tracks which client (`TypeProvider`) will carry this operation. Or we should call it `owner`?